pkg/controller/provider/container/ocp/collection.go

```diff
@@ -65,7 +70,6 @@ func (r *StorageClass) Reconcile() (err error) {
 //
 // Resource created watch event.
 func (r *StorageClass) Create(e event.CreateEvent) bool {
-	Log.Reset()
```

Comment:
The Reset() here in a different coroutine is no longer necessary and defeats the purpose of the correlation ID in the log entries like: "provider|<reconcile-id>" ...